### PR TITLE
fixed pbs core ranges feature

### PR DESCRIPTION
**Pull request by sfranky**

### CODE REVIEW

File: plugins/pbs.py [hunk 1]
```diff
@@ -218,15 +220,19 @@ def _get_jobs_cores(jobs):  # block['jobs']
...
-            if len(core) > len(job):  # PBS vs torque?
-                core, job = job, core
-            job = job.strip().split('/')[0].split('.')[0]
-            yield job, core
+            if (',' in core) or ('-' in core):
+                for (subcore, subjob) in PBSBatchSystem.get_corejob_from_range(core, job):
+                    subjob = subjob.strip().split('/')[0].split('.')[0]
+                    yield subjob, subcore
+            else:
+                if len(core) > len(job):  # PBS vs torque?
```
Comment (fgeorgatos) [inline plugins/pbs.py]: do you mean PBSpro?

Comment (sfranky) [inline plugins/pbs.py]: no, this is old and pre-existing

**Comment by fgeorgatos** [comment]
all looks good, let's merge

File: plugins/pbs.py (hunk 1 above)
Comment (fgeorgatos) [inline plugins/pbs.py]: ok!

**Pull request merged**